app/__tests__/dashboard.test.ts

```diff
@@ -1,21 +1,4 @@
 import { shortest } from "@antiwork/shortest";
-import { clerk, clerkSetup } from "@clerk/testing/playwright";
-
-let frontendUrl = process.env.SHORTEST_TEST_BASE_URL ?? "http://localhost:3000";
-
-shortest.beforeAll(async ({ page }) => {
-  await clerkSetup({
-    frontendApiUrl: frontendUrl,
-  });
-  await clerk.signIn({
-    page,
-    signInParams: {
-      strategy: "email_code",
-      identifier: "shortest+clerk_test@example.com",
-    },
-  });
-
-  await page.goto(frontendUrl + "/dashboard");
-});
 
+shortest("Login to the App using magic link. Use this email: 'stay-ants@taoyeoeo.mailosaur.net'");
```

Comment:
Removing injecting session into browser and instead using magic link. Because this would perform email validation as well as login validation. 

Here's a demo: 


https://github.com/user-attachments/assets/160f3183-49fe-406c-a781-6124cf77676c